src/components/Products/productCard.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import { CapitalizeString } from '../../utils/utils';
+import {
+  CardsStyledWrapper,
+  StyledContainerImage,
+  StyledPhoto,
+  Title,
+  Category,
+  Price,
+  SeeDetails,
+  AddToCart
+} from './productCardComponents'
+
+const ProductCard = ({ data, id }) => {
+  const navigate = useNavigate();
+
+  const SendToCart = () => {
```

Comment:
Normally the functions that passed to the onClick event or whatever event start with the `handle` word. This is only a convention. So here would be `handleSendToCart`